addons/html_editor/static/src/others/embedded_component_plugin.js

```diff
@@ -182,6 +182,10 @@ export class EmbeddedComponentPlugin extends Plugin {
         if (StateChangeManager) {
             env.getStateChangeManager = this.getStateChangeManager.bind(this);
         }
+        for (const cb of this.resources["embedded_component_env"]) {
+            // Add plugin specific additional environments
+            Object.assign(env, cb());
```

Comment:
in the second argument of `mountComponent`, you should add `name`, and provide the name to the `cb`
=> this will allow to provide the extra env only to components that use it